rosetta-lang/src/main/java/com/regnosys/rosetta/generator/java/types/JavaPojoProperty.java

```diff
@@ -127,6 +127,21 @@ protected void appendTo(TargetStringConcatenation target) {
 		}, type);
 	}
 	
+	//Requires review: Safe guard for NullPointerException on multi-cardinality null properties
+	public JavaExpression applyGetterWithNullableWrapper(JavaExpression expr) {
```

Comment:
Hm, I would like to keep the `JavaPojoProperty` general for any property, whereas this method assumes it is multicardinality. See a more direct suggestion in the `ValidatorsGenerator`.